tests/e2e/testfiles/scenarios.yaml

```diff
@@ -1,69 +1,53 @@
-# List of E2E Tests Scenarios
-
-- scenario: "Install Authorization Proxy Server V1"
+#List of E2E Tests Scenarios
+- scenario: "Install Authorization Proxy Server"
```

Comment:
The authorization scenarios were updated recently in PR #641. Please maintain those changes to not introduce breaking changes to the authorization scenarios. Thank you.